ion-schema/src/main/kotlin/com/amazon/ionschema/internal/TypeCore.kt

```diff
@@ -36,9 +37,7 @@ internal class TypeCore(
         else -> IonType.valueOf(nameSymbol.stringValue().toUpperCase())
     }
 
-    private val ionTypeName = ionType.schemaTypeName()
```

Comment:
This change is incidental. I noticed that the `ionTypeName` variable is completely redundant (it's a private variable that is always the same as `name`), so I removed it.